phases/04-computer-vision/16-vision-pipeline-capstone/code/main.py

```
@@ -0,0 +1,210 @@
+        return list(zip(cls.tolist(), scores.tolist()))
+
+    def run(self, image, image_id="anonymous"):
+        t0 = time.perf_counter()
+        tensor = self.preprocess(image)
+        det = self.detect(tensor)
+
+        crops = []
+        valid_indices = []
+        detections = []
+        for i, (box, score, cls) in enumerate(
+            zip(det["boxes"], det["scores"], det["labels"])
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)